src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/__tests__/IncidentSplitForm.test.js

```diff
@@ -70,6 +69,8 @@ describe('IncidentSplitForm', () => {
       department: parentIncidentFixture.directingDepartment,
       // eslint-disable-next-line no-sparse-arrays
       incidents: [
+        ,
+        ,
```

Comment:
Why is this necessary?

Reply:
Not fully sure, but something is wrong in this component. It adds empty entries in the array...